packages/ui-components/src/components/Logo/Logo.tsx

```diff
@@ -26,10 +26,10 @@ interface Props {
 
 const Logo: React.FC<Props> = ({ size, onClick, className, isDefault = false }) => {
   const { configOptions } = useConfig();
-  if (!isDefault && configOptions?.logo) {
+  if (!isDefault && (configOptions?.logo || configOptions?.logoURI)) {
```

Comment:
🤔 from where logoURI is coming from? I don't find it in the source code. Also seems not exposed 

https://github.com/verdaccio/verdaccio/blob/master/packages/middleware/src/middlewares/web/utils/renderHTML.ts

```
 const options: TemplateUIOptions = {
    showInfo,
    showSettings,
    showThemeSwitch,
    showFooter,
    showSearch,
    showDownloadTarball,
    darkMode,
    url_prefix,
    basename,
    base,
    primaryColor,
    version,
    logo,
    flags,
    login,
    pkgManagers,
    title,
    scope,
    language,
  };
````